RealtimeStreamToImpala/config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "template":"etlBatch",
```

Comment:
actually it was updated already this morning unless we're talking about different files